this unconditionally uses broadwell/ryzen acx instructions in curve25519-inline.h
--
--- a/nss/lib/freebl/freebl.gyp
+++ b/nss/lib/freebl/freebl.gyp
@@ -935,7 +935,7 @@
     'module': 'nss',
     'conditions': [
       [ 'target_arch=="x64" and cc_is_gcc==1', {
-        'supports_vale_curve25519%': 1,
+        'supports_vale_curve25519%': 0,
       }, {
         'supports_vale_curve25519%': 0,
       }],
